doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -140,9 +144,13 @@ Testing steps
 
 To perform a FOTA update, complete the following steps:
 
-1. Create a binary file that contains the new image.
+.. fota_upgrades_over_ble_nrfcdm_common_dfu_steps_start
 
-   |fota_upgrades_building|
+1. Generate the DFU package by building your application with the FOTA support over Bluetooth Low Energy.
+   You can find the generated DFU archive in the following location :file:`<build_dir>/zephyr/dfu_application.zip`.
+
+   .. note::
+      Use a higher version for each image included in the DFU package as the FOTA process may be cancelled due to the downgrade prevention.
```

Comment:
I guess I was too tired when composing :)

```suggestion
      For each image included in the DFU-generated package, you should use a higher version number than your currently active firmware, as your DFU target may reject the FOTA process due to a downgrade prevention mechanism.
```

Reply:
@annwoj, please take a look also at my suggestion

Reply:
Sounds good, though if we're going with this description, I'd split it into 2 sentence and do some minor adjustments, e.g.,:
For each image included in the DFU-generated package, use a higher version number than your currently active firmware. Otherwise, the DFU target may reject the FOTA process due to a downgrade prevention mechanism.

Reply:
Sure, thanks for feedback